### DISCUSSION TITLE
ORtools error

### DISCUSSION THREAD

**Discussion by wanghaijian217** [ORTools4MTEX/ORTools#42] [Q&A] [answered]
Hello Frank and Azdiar, when I run example7 directly, the following error report shows that too many parameters have been entered, which is reported in line 36 of computeVariantGrains and displayed in line 159 of example7.
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/acfbc5de-bc08-4a46-a9a5-3e50a1642f68)
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/1a701028-c153-44dc-a27a-05545fa94a29)

**Comment by AzdiarGazder** [comment, marked as answer]
Hi Long,

We are in the process of minor releases with new versions of both, MTEX and ORTools.
One of the additions in the new versions is the ability to compute Bain group Ids in both, MTEX and ORTools.

To prevent the error, and until the stable versions are officially released, please use the developer versions of  both, MTEX and ORTools.

Warm regards,
Azdi
